packages/sku/scripts/test.js

```diff
@@ -7,17 +6,15 @@ const { argv } = require('../config/args');
 const { runVocabCompile } = require('../lib/runVocab');
 
 const log = debug('sku:jest');
-// https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#configuring-your-testing-environment
-process.env.IS_REACT_ACT_ENVIRONMENT = true;
```

Comment:
According to the linked docs, this never worked? It needs to be set on `globalThis`, not `process.env`, so I'm unsure what this was accomplishing, if anything.